packages/services/f-http/src/tests/errorHandler.test.js

```diff
@@ -0,0 +1,38 @@
+import handleError from '../errorHandler';
+
+describe('errorHandler', () => {
+    it('errorHandler should be defined', () => {
+        // Arrange, Act & Assert
+        expect(handleError).toBeDefined();
+    });
+
+    it('errorHandler should use callback when configured', () => {
+        // Arrange
+        const error = {
+            message: 'This is an example error'
+        };
+
+        const errorCallbackMock = jest.fn();
+
+        // Act
+        handleError(error, errorCallbackMock);
+
+        // Assert
+        expect(errorCallbackMock.mock.calls.length).toBe(1);
```

Comment:
You can get rid of this now.

Reply:
I guess I still want to make sure it is just called once

Reply:
I might be wrong, but I think the `toHaveBeenCalledWith();` will fail if it's called more than once. If I'm wrong, you can replace your line with `expect(errorCallbackMock).toHaveBeenCalledTimes(1);` which in my opinion looks slightly nicer and more readable.

Reply:
Ill go with the latter; prefer it to be obvious if not from the way the test functions, but how it reads.